persistence/device.go

```diff
@@ -49,6 +53,7 @@ type ExchangeDevice struct {
 	Org                string      `json:"organization"`
 	Pattern            string      `json:"pattern"`
 	Name               string      `json:"name"`
+	NodeType           string      `json:"nodeType"`
```

Comment:
If an old device is upgraded with this code, this field will be empty. I think we need should add code to SyncOnInit() to set the nodeType to device. We need to do the same thing for the exchange resource.

Reply:
Will change the code in the db read part, to assign it to "device" if is empty.

Reply:
done